api/v1/models/Attendee.js

```diff
@@ -13,11 +13,11 @@ const ACCEPTANCE_TYPES = ['CREATE', 'CONTRIBUTE'];
 const Model = require('./Model');
 const Upload = require('./Upload');
 const AttendeeProjectInterest = require('./AttendeeProjectInterest');
-const AttendeeProject = require('./AttendeeProject');
 const AttendeeExtraInfo = require('./AttendeeExtraInfo');
-const AttendeeEcosystemInterest = require('./AttendeeEcosystemInterest');
 const AttendeeRequestedCollaborator = require('./AttendeeRequestedCollaborator');
 const AttendeeRSVP = require('./AttendeeRSVP');
+const AttendeeWebsite = require('./AttendeeWebsite');
```

Comment:
I'd rather use the existing relational ExtraInfo model for additional details they might want to add, and then introduce a new LongForm model for their srq

Reply:
New ExtraInfo and LongForm models created in b7fb5a6707ba789a9ce989f74461bd975cd1807b and 4dbfa1b32bc2b1277c64598623b3875edbcfa8f1